server/server.yml

```diff
@@ -38,6 +38,16 @@
 #
 # firebase-key-file: <filename>
 
+# Enable web push
+#
+# Run ntfy web-push generate-keys to generate the keys
+#
+# web-push-enabled: true
+# web-push-public-key: ""
+# web-push-private-key: ""
+# web-push-subscriptions-file: ""
+# web-push-email-address: ""
```

Comment:
- This needs to be a little more verbose. Usually this section is largely identical to whatever I put in the config.md section on the topic.
- The commented values should match the defaults, so in this case:

```yaml
# web-push-enabled: false
# web-push-public-key: 
# web-push-private-key: 
# web-push-subscriptions-file: 
# web-push-email-address: 
```